<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newattendance</class>
 <widget class="QMainWindow" name="newattendance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_newattendance">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1261</width>
      <height>1001</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 189, 182);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>100</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Fy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ty</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BE</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lbl_class">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Class&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_Rollno">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Roll No&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_month">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>240</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Month&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>240</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>January</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>February</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>March</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>April</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>May</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>June</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>July</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>August</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>September</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Octomber</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>November</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>December</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_Totallec">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>320</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Total Lectures&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_attentedlecture">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>390</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Attentend Lectures&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>480</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;Serif&quot;;
font: 15pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>170</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_totalLectures">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>310</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;40&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_Atteentedlect">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>380</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_newattendancetitle">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 15pt &quot;Serif&quot;;
selection-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ef2929;&quot;&gt;NEW ATTENDANCE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
